.github/workflows/mobilecoin-dev-cd.yaml

```
@@ -279,7 +275,7 @@ jobs:
 
 #########################################
 # Build/Publish public artifacts
-#########################################
+######################################### 1a94367
```

:upside_down_face: guessing this was some  left over debug code
```suggestion
#########################################
```